### PR TITLE
Ft/Featured stories

### PR DESCRIPTION
**Pull request by kelvinkipruto**
## Description
This PR implements the Featured stories block

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Screenshots
image

image

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: apps/codeforafrica/src/payload/blocks/PostsOverView.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+      required: true,
+      type: "text",
+    },
+    {
+      name: "primaryTag",
+      label: "Post Type",
+      required: true,
+      type: "select",
+      options: primaryTags,
+      hasMany: false,
```
Comment (kilemensi) [inline apps/codeforafrica/src/payload/blocks/PostsOverView.js:31]: I think this block is only for stories... we don't have a featured view for opportunities in the designs.

File: apps/codeforafrica/src/payload/blocks/PostsOverView.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+      required: true,
+    },
+    {
+      name: "posts",
+      type: "relationship",
+      relationTo: "posts",
+      hasMany: true,
+      minRows: 3,
+      maxRows: 3,
```
Comment (kilemensi) [inline apps/codeforafrica/src/payload/blocks/PostsOverView.js:46]: These needs to be posts of type stories only.

You can also make this a group as well
```js
stories: {
  items: // This is the relationship field
  action: // This is the action linkGroup field
}
```

**Review by kilemensi — COMMENTED** [review]
👍🏽 

---

I think lets call the block NewsAndStories as well (`news-stories` slug).

File: apps/codeforafrica/src/payload/blocks/FeaturedStories.js [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+      en: "Featured Stories",
+    },
+    plural: {
+      en: "Featured Stories",
+    },
+  },
+  fields: [
+    {
+      name: "title",
+      label: "Title",
```
Comment (kilemensi) [inline apps/codeforafrica/src/payload/blocks/FeaturedStories.js:18]:
```suggestion
```

If label is the same as title, payload will user name. No need to add it.

**Review by kilemensi — APPROVED** [review]

**Review by koechkevin — APPROVED** [review]

**Pull request merged**